README.md

```diff
@@ -1,44 +1,37 @@
-!['Device Posture API logo'](https://raw.githubusercontent.com/w3c/device-posture/gh-pages/images/sf-logo-s.png)
+<img width="247" alt="favicon" src="https://user-images.githubusercontent.com/73939538/123794912-4c089380-d8db-11eb-811b-7367461cfdd4.png">
 
 # Device Posture API
 
 ## Authors
 * [Diego González-Zúñiga](https://twitter.com/diekus) ([Samsung](https://samsunginter.net))
 * [Kenneth Rohde Christiansen](https://twitter.com/kennethrohde) ([Intel](https://intel.com))
 
-## Explainer
-* [Explainer on GitHub](https://github.com/SamsungInternet/Explainers/blob/gh-pages/Foldables/FoldState.md)
-
 ## Abstract
 
- This document specifies an API that allows web applications to request and be notified of changes of the posture of a foldable device.
+This document specifies an API that allows web applications to request and be notified of changes of the posture of a device.
 
 ## Goals
 
-Foldable devices come in many shapes and sizes. While the use cases for the ultrabooks of the past vary significantly from those of the new trend of mobile devices due to the inherent focus on screen real estate, **the data in question is similar since it’s related to the angle of the fold**.
-
-The device posture is the physical position in which a device holds which may be derived from sensors in addition to the angle. The **main interest in knowing the device posture is because there are interesting opportunities in the area of responsive design that enable new user experiences**. With these new devices, the user can choose to consume content and browse the web even when the device is not flat, in which case the developer might want to provide a different layout for the content depending on how the device is being used. We propose a way to expose information about the posture of the device to the developer.
```

Comment:
Please leave the definition of device posture since it's something that has been discussed and agreed to add.
"The device posture is the physical position in which a device holds which may be derived from sensors in addition to the angle."

Reply:
@lauramorinigo Thanks! I will add the definition again, good catch!

Reply:
@kenchris pls feel free to merge so I can add: "The device posture is the physical position in which a device holds which may be derived from sensors in addition to the angle."